### PR TITLE
Also print list of stories

### PR DESCRIPTION
**Pull request by q-wertz**
Fixes https://github.com/nextcloud/calendar_resource_management/issues/33

First lines of code in php. Hope it is valid. Do there have to be any changes to unittests?

### CODE REVIEW

File: lib/Command/ListResources.php [hunk 1]
```diff
@@ -114,6 +116,32 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
+		$table->setHeaders(
+			[
+				'ID',
+				'Located in',
+				'Display Name'
+			]
+		);
+		foreach ($buildings as $building) {
+			$stories = $this->storyMapper->findAllByBuilding($building->getId());
+			/** @var StoryModel $story */
```
Comment (q-wertz) [inline lib/Command/ListResources.php:131]: Not sure if this line is needed

File: lib/Command/ListResources.php [hunk 2]
```diff
@@ -114,6 +116,32 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
+			foreach ($stories as $story) {
+				$table->setRow($row,
+					[
+						$story->getId(),
+						$building->$building->getDisplayName(),
```
Comment (miaulalala) [inline lib/Command/ListResources.php:136]:
```suggestion
						$building->getDisplayName(),
```

Comment (q-wertz) [reply lib/Command/ListResources.php]: Thanks for noticing :sweat_smile:

**Comment by q-wertz** [comment]
Fixed the error @miaulalala reviewed. Should be ready now for merging.

**Review by miaulalala — APPROVED** [review]

**Comment by miaulalala** [comment]
@q-wertz Thanks for your contribution!

**Pull request merged**